Formula/logrotate.rb

```diff
@@ -1,8 +1,8 @@
 class Logrotate < Formula
   desc "Rotates, compresses, and mails system logs"
   homepage "https://github.com/logrotate/logrotate"
-  url "https://github.com/logrotate/logrotate/releases/download/3.17.0/logrotate-3.17.0.tar.xz"
-  sha256 "58cc2178ff57faa3c0490181cce041345aeca6cff18dba1c5cd1398bf1c19294"
+  url "https://github.com/logrotate/logrotate/releases/download/3.18.0/logrotate-3.18.0.tar.xz"
+  sha256 "841f81bf09d0014e4a2e11af166bb33fcd8429cc0c2d4a7d3d9ceb3858cfccc5"
   license "GPL-2.0"
```

Comment:
```suggestion
  license "GPL-2.0-or-later"
```
?

Ref. https://github.com/logrotate/logrotate/blob/a7f57af13326f196dc1c8420ff06929458f33d9b/logrotate.spec.in#L5